README.md

```diff
@@ -39,11 +39,13 @@ Submitting a PR is as simple as following these steps:
 
 3. **Edit the `.ts` file:** Locate and edit the applicable `.ts` file inside the `config` folder. Make sure to provide all the necessary information about your project.
 
-4. **Run configuration:** Run `yarn configure` to convert your TypeScript entry into a consumable JSON file.
+4. **Add your logo** Add your token's logo in .svg or .png format. The size of your logo should be 200 x 200 pixels, less than 50 KB and in the form of a circle.
```

Comment:
Let's have this say something more like "should be square or circular and at least 200 x 200 pixels if PNG, ideally under 50KB"